main_server.py

```
@@ -208,10 +208,30 @@ async def _handle_agent_event(event: dict):
+                    "task_id": event.get("task_id") or "",
+                    "channel": event.get("channel") or "unknown",
+                    "status": cb_status,
+                    "success": bool(event.get("success", True)),
+                    "summary": event.get("summary") or text,
+                    "detail": event.get("detail") or text,
+                    "error_message": event.get("error_message") or "",
+                    "timestamp": event.get("timestamp") or "",
+                }
+                mgr.enqueue_agent_callback(callback)
+                # Attempt immediate delivery; re-queued automatically if session is busy
+                asyncio.create_task(mgr.trigger_agent_callbacks())
```

Ruff (0.15.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)